demos/storybook/stories/info-list-item/with-full-config.stories.ts

```diff
@@ -18,13 +18,15 @@ export const withFullConfig = (): any => ({
             (click)="action()">
             <div pxb-title>{{title}}</div>
             <div pxb-subtitle>{{subtitle}}</div>
+            <div pxb-info>{{info}}</div>
```

Comment:
So in Angular we added this knob in the full-config story and in React it's has its own story...we shuld be consistent.  I kind of like it in the 'basic' story, but if we'd rather try to keep people to just two lines in typical usage, then full-config probably makes more sense.